src/fiatExchanges/PaymentMethodSection.test.tsx

```diff
@@ -121,5 +121,40 @@ describe('PaymentMethodSection', () => {
     const infoElement = queryByTestId('Bank/provider-0/info')
     expect(infoElement).toBeTruthy()
     expect(infoElement).toHaveTextContent('selectProviderScreen.numDays')
+    expect(infoElement).not.toHaveTextContent('selectProviderScreen.idRequired')
+  })
+
+  it.each([
+    [
+      PaymentMethod.Card as const,
+      normalizeQuotes(CICOFlow.CashIn, [], [mockProviders[2]]),
+      'card',
+      'oneHour',
+    ],
+    [
+      PaymentMethod.Bank as const,
+      normalizeQuotes(CICOFlow.CashIn, [mockFiatConnectQuotes[1]] as FiatConnectQuoteSuccess[], []),
+      'bank',
+      'numDays',
+    ],
+    // TODO(any): enable as part of ACT-518
```

Comment:
yeah, the problem with testing this is that we can't create a "FiatConnectQuote" object with "MobileMoney" since it's not supported. I guess we could create a mock object that resembles a "FiatConnectQuote" object, but I am not sure if it's worth it since this is going to be supported pretty soon. Any other ideas for testing this now?